pkg/azure/plugin_integration_test.go

```diff
@@ -410,12 +412,28 @@ func TestAttachResourceIntegration(t *testing.T) {
 		ResourceGroupName: resourceGroupName,
 		ResourceName:      externalVmName,
 	}
-	externalAddressSpace := "20.9.0.0/24"
+
+	// Find unused address spaces for external address
+	conn, err := grpc.NewClient(azureServer.orchestratorServerAddr, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		utils.Log.Printf("Azure Integration test: Could not dial the orchestrator")
+		return
+	}
+	defer conn.Close()
+	client := paragliderpb.NewControllerClient(conn)
+	reqAddressSpaces := make([]int32, 2)
```

Comment:
Why length of 2? I thought we only need one external address space, in which we can actually pass `nil`.

Reply:
You're right. We do need only 1. I had 2 for debugging purposes. Fixed this now. Thanks Sean